build/ci/.azure-devops-package-netcoremobile.yml

```diff
@@ -52,6 +52,7 @@ jobs:
   steps:
   - checkout: self
     clean: true
+    fetchTags: false
```

Comment:
Let's keep the tags for all package builds, even if gitversion uses a cache at this time.